sci-electronics/petrify/petrify-4.2-r1.ebuild

```diff
@@ -1,35 +1,33 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=8
 
 DESCRIPTION="Synthesize Petri nets into asynchronous circuits"
 HOMEPAGE="http://www.lsi.upc.edu/~jordicf/petrify/"
 SRC_URI="http://www.lsi.upc.edu/~jordicf/petrify/distrib/petrify-4.2-linux.tgz"
+S="${WORKDIR}"/${PN}
 
 LICENSE="Old-MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
 
 RDEPEND="media-gfx/graphviz"
-DEPEND=""
+DEPEND="${RDEPEND}"
 
 RESTRICT="strip"
 
 QA_PREBUILT="/opt/petrify/petrify"
```

Comment:
Relative paths in QA_* (check this is documented sufficiently?)

Reply:
re: doc, fwiw it's mentioned under these variables in [ebuild(5)](https://devmanual.gentoo.org/eclass-reference/ebuild/index.html), which is where one would be looking if looking into these afaik. I guess if the docs had an example it may be easier to pick up though.

Reply:
I think we should really have these in the devmanual in a proper section too, but not sure.

Reply:
Yeah, I still see absolute paths often so I wasn't sure. But I agree that we should have examples somewhere